index-handlers/src/main/java/no/sikt/nva/nvi/index/model/NviCandidateIndexDocument.java

```diff
@@ -16,7 +17,8 @@ public record NviCandidateIndexDocument(@JsonProperty(CONTEXT) URI context,
                                         String identifier,
                                         PublicationDetails publicationDetails,
                                         List<Approval> approvals,
-                                        int numberOfApprovals) {
+                                        int numberOfApprovals,
+                                        BigDecimal points) {
```

Comment:
I think I would call this totalPoints to prevent confusion between totalPoints and points per institution

Reply:
This object does not have other points, I think it is a bit misleading to call it totalPoints when we dont know/dont have points?